packages/frontend/src/translations/en.global.json

```
@@ -1738,5 +1741,33 @@
             "noWithdraw": "Unable to withdraw pending balance from validator"
         }
     },
-    "warning": "Warning"
+    "warning": "Warning",
+    "zeroBalance": {
+        "ledgerModal": {
+            "desc": "Add a Ledger limited access key to the wallet to manage multiple recovery methods.",
+            "confirmOnLedger": "Confirm the action on your Ledger device.",
+            "title": "Your account is now active!",
+            "addLedgerKey": "Add Ledger Key"
+        },
+        "addLedgerKey": {
+            "success": {
+                "header": "Ledger access key added successfully!",
+                "message": "A new Ledger key was added to your account. You can now manage your accounts recovery methods using your Ledger device."
```

```suggestion
                "message": "A new Ledger key was added to your account. You can now manage your accounts recovery methods without using your Ledger device."
```
Might be wrong on this - is this messaging based around the limited access key we add for authenticating requests to contract-helper? If so, then it would enable users to managing recovery methods _without_ their ledger?

---

It's just so that they can pull the information from the contract-helper, not actually manage the method, but figured it's easier for user to understand 'manage' in this context